modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -28,16 +26,32 @@ If no external IdPs are enabled, Access Manager acts as the default authorizatio
 
 == The Use Case
 
-To better understand how to implement multiple external credential providers, imagine a company, "Good Weather" that uses meteorology to forecast changing weather. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
+To better understand how to implement multiple external client providers, imagine a company, "Good Weather" that uses meteorology to forecast changing weather. Good Weather collects quantitative data about the current state of the atmosphere at a given place and, using meteorology, predicts how that current state will change over time.
 
 Good Weather has several sub-organizations running under the master organization. These sub-organizations exist based on the functions they perform, for example, Engineering and Forecasting. The Forecasting department researches and collects recent meterological data from various sources. This data is then consumed by various client applications. While the Engineering department maintains the applications that host the forecasting research data.
 
 Because Good Weather must ensure that its weather-forecasting data is accurate and secure, the weather APIs are accessed only by employees of the Forecasting department. Good Weather also wants to ensure that the production APIs are the single source of truth at all times.
 
 To ensure security and access segregation for different organizations, environments, and APIs, Good Weather IT sets up separate external client providers for the Forecasting department.
 
-//example of how they implement the client providers.
-//Tomas how will they best implement this?
+== Onboarding Multiple Client Providers
+
+Your onbaording multiple client providers procedure differs based on how you have configured client providers and your business organizations in your company:
+
+* *Scenario 1: No external providers configured:*
++
+Creating and modifying such environments and APIs behave the same. The native Anypoint Platform client provider is assigned for such environments. The existing APIs and environment have no impact.
+* *Scenario 2: External client provider configured:*
++ 
+If you have external client providers, such as PingFederate, configured for your environments, the client provider is automatically ported when you onboard the multiple client providers feature. However, when you create a new environment, you must manually assign the correct client provider to your environment in Access Management. 
```

Comment:
```suggestion
If you have external client providers such as PingFederate configured for your environments, the client provider is automatically ported when you onboard the multiple client providers feature. However, when you create a new environment, you must manually assign the correct client provider to your environment in Access Management. 
```
HOWEVER...are you onboarding the FEATURE or the actual client providers?

Reply:
No the feature. Thanks! Done!